src/modules/settings/components/Settings.jsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
-// eslint-disable-next-line import/extensions, import/no-unresolved
-import './settings/*';
 import Panel from 'rsuite/Panel';
 import {Icon} from '@rsuite/icons';
 import InputGroup from 'rsuite/InputGroup';
 import AutoComplete from 'rsuite/AutoComplete';
 import * as faSearch from '@fortawesome/free-solid-svg-icons/faSearch';
-import {Components} from './Store.jsx';
 import FontAwesomeSvgIcon from '../../../common/components/FontAwesomeSvgIcon.jsx';
 
-const settings = Object.values(Components).sort((a, b) => a.name.localeCompare(b.name));
+let settings = [];
+
+export async function importStoreCallback() {
```

Comment:
instead of exporting this, can we instead use `useEffect` to load this below?

```
let settings = null;
async function loadSettings() {
  if (settings != null) {
    return settings;
  }
  const {Components} = await import('./Store.jsx');
  settings = Object.values(Components).sort((a, b) => a.name.localeCompare(b.name));
}

...

const [settings, setSettings] = useState([]);
useEffect(() => {
  loadSettings.then(settings => setSettings(settings));
});
```